.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
Please remove everything under each of the headings as it's just extra content that whoever opens an issue will need to remove themselves. Each section should be the heading followed by two empty lines:

```md
**Heading**


**Another heading**
```